services/how-to/test-net.md

```diff
@@ -0,0 +1,33 @@
+Testnet tokens allow you to simulate transactions, deploy smart contracts, and interact with dapps in a controlled environment.
+
+:::note
+
+Use [testnet endpoints](../get-started/endpoints.md) during development and initial testing phases before moving to mainnet for production deployment. 
+
+:::
+
+
+| Testnet     | Faucet URL        | Bridge URL                    |                             
+|-------------|-------------------|-------------------------------|
+| [Arbitrum Sepolia](https://docs.arbitrum.io/launch-orbit-chain/how-tos/add-orbit-chain-to-bridge-ui)| [Arbitrum Sepolia faucet](https://faucet.quicknode.com/arbitrum/sepolia)| [Arbitrum bridge](https://bridge.arbitrum.io/)|
+| [Avalanche Fuji](https://docs.avax.network/build/tutorials/platform/fuji-workflow)| [Avalanche Fuji faucet](https://faucet.avax.network)| [Avalanche bridge](https://bridge.avax.network/)|
+| [Base Sepolia](https://docs.base.org/docs/using-base/)| [Base Sepolia faucet](https://faucet.quicknode.com/base/sepolia)| [Base bridge](https://bridge.base.org/deposit)|
+| [Blast Sepolia](https://docs.blast.io/using-blast)| [Blast Sepolia faucet](https://faucet.quicknode.com/blast/sepolia)|     |
+| [Binance Smart Chain Sepolia (BSC)](https://academy.binance.com/en/articles/connecting-metamask-to-binance-smart-chain)| [BSC Sepolia faucet](https://www.bnbchain.org/en/testnet-faucet)| [BSC bridge](https://testnet.bscscan.com/bridge)|
+| [Celo Alfajores](https://docs.celo.org/getting-started/alfajores-testnet)| [Celo Alfajores faucet](https://faucet.celo.org/alfajores)| [Celo bridge](https://docs.celo.org/protocol/bridge#token-bridges)|
+| [Ethereum Sepolia](https://docs.blast.io/building/bridges/testnet)| [Ethereum Sepolia faucet](https://faucet.quicknode.com/ethereum/sepolia)| [Sepolia bridge](https://docs.blast.io/building/bridges/testnet)|
+| [Linea Sepolia](https://support.linea.build/getting-started/how-to-get-linea-goerli-testnet-eth)| [Linea Sepolia faucet](https://www.infura.io/faucet/linea) or [Covalent faucet](https://goldrush.dev/faucet/)| [Linea bridge](https://bridge.linea.build)|
+| [Mantle Sepolia](https://docs.mantle.xyz/network/for-devs/developing-on-mantle#connecting-to-mantle-testnet)| [Mantle Sepolia faucet](https://faucet.sepolia.mantle.xyz) |[Mantle bridge](https://bridge.sepolia.mantle.xyz)| 
+| [Optimism Sepolia](https://community.optimism.io/docs/developers/tutorials.html)| [Optimism Sepolia faucet](https://faucet.quicknode.com/optimism/sepolia)| [Optimism bridge](https://app.optimism.io/bridge)| 
+| [Palm](https://docs.palm.io/get-started/connect/testnet)| [PALM token](https://docs.palm.io/get-started/tokens)| [Token bridge](https://app.palm.io/bridge)|
+| [Polygon Amoy](https://docs.polygon.technology/tools/wallets/metamask/add-polygon-network/)| [Polygon Amoy faucet](https://faucet.polygon.technology)| [Polygon bridge](https://bridge.polygon.technology/)| 
```

Comment:
The bridge link doesn't work. Is this an appropriate replacement? https://portal.polygon.technology/bridge